### ISSUE TITLE
Elasticsearch: Support nested property names in field auto completion 

### ISSUE THREAD

**Issue by torkelo** [grafana/grafana#3772]

**Comment by arcolife** [comment]
Doesn't PR https://github.com/grafana/grafana/pull/4527/ fix this? Thought of connecting it here.
@torkelo the tests in that PR have passed apparently. Did you make some changes to timestamps tests?

**Comment by torkelo** [comment]
@arcolife yes, that PR does that. not sure what your referring to about tests / timestamp. 

the reason it is not merged is that it has no unit tests.

**Comment by portante** [comment]
@arcolife, do you have a plan for adding unit tests here?

**Comment by arcolife** [comment]
@portante I had added unit tests later on. Tracked through https://github.com/grafana/grafana/pull/4527 and finally through https://github.com/grafana/grafana/pull/4694#issuecomment-215995978

**Comment by torkelo** [comment]
Sorry, stale bot was wrongly configured (should only have acted on PRs)

**Comment by aocenas** [comment]
Hey @torkelo, can you add some details for this, is this still relevant today?

**Comment by torkelo** [comment]
Not sure, is it already implemented?

**Comment by aocenas** [comment]
@Elfo404 can you take a look if this is still an issue seem like there were some PRs already merged for this so it's probably already fixed.

**Comment by Elfo404** [comment]
Took some time to look at this and better understand the context. Apparently it received some interest over time (although not a lot compared to other feature requests).

I'm leaving the "needs investigation" label as I'm not how tricky it would be to implement and I'm not sure about the priority we should assign to this.
